libs/designer/src/lib/core/actions/bjsworkflow/operationdeserializer.ts

```diff
@@ -190,9 +189,7 @@ export const initializeOperationDetailsForManifest = async (
         manifest,
         undefined /* swagger */,
         operation,
-        rootNodeId,
-        connectionReferences,
-        dispatch
+        rootNodeId
```

Comment:
Does this not require any changes in your settings file?

Reply:
It is already added as an optional property